Standard.AI.OpenAI/Models/Services/Foundations/ExternalFineTunes/ExternalHyperparams.cs

```diff
@@ -0,0 +1,23 @@
+﻿// ---------------------------------------------------------------------------------- 
+// Copyright (c) The Standard Organization, a coalition of the Good-Hearted Engineers 
+// ----------------------------------------------------------------------------------
+
+using Newtonsoft.Json;
+
+namespace Standard.AI.OpenAI.Models.Services.Foundations.ExternalFineTunes
+{
+    public class ExternalHyperparams
```

Comment:
This should be internal.

```suggestion
    internal class ExternalHyperparams
```